gulp/tasks/development/build.js

```diff
@@ -5,9 +5,8 @@ var runSequence = require('run-sequence');
  * Run all tasks needed for a build in defined order
  */
 gulp.task('build', function(callback) {
-  runSequence('delete',
+  runSequence('delete', 'jekyll',
```

Comment:
@OZARIN: I guess this isn’t necessary, as it shouldn’t matter if `jekyll` is run in parallel after `delete` or in sequence with the other tasks after `delete`.